src/main/proto/firrtl.proto

```diff
@@ -0,0 +1,451 @@
+syntax = "proto3";
+
+package hardware.chips.chisel3;
```

Comment:
Should change package, maybe to "proto"? Not sure...

Reply:
Done.